lib/driver.js

```diff
@@ -137,9 +137,20 @@ const MEMOIZED_FUNCTIONS = [
   'getScreenInfo',
 ];
 
+const ARGS_CONSTRAINTS = {
+  webkitDebugProxyPort: {
+    isNumber: true
+  },
+  wdaLocalPort: {
+    isNumber: true
+  },
+};
+
 class XCUITestDriver extends BaseDriver {
-  constructor (opts = {}, shouldValidateCaps = true) {
-    super(opts, shouldValidateCaps);
+  constructor (opts = {}, shouldValidateCaps = true, driverArgs = {}) {
+    opts = parseServerArgs(opts, driverArgs, ['webkitDebugProxyPort', 'wdaLocalPort'], ARGS_CONSTRAINTS);
```

Comment:
let's move the array (3rd arg) to a named constant just like we do with ARGS_CONSTRAINTS

Reply:
we could just use Object.keys(ARGS_CONSTRAINTS)